faq: add steps to re-register a board (#397)
foundriesio/docs: source/reference-manual/troubleshooting/troubleshooting.rst

After these changes, the files ``/usr/lib/passwd`` and ``/usr/lib/group`` should
include the configuration for the new user.

Re-register a Device
^^^^^^^^^^^^^^^^^^^^

> It may be needed to re-register the same device during development. In this case,

Reviewer: ```suggestion
You may need to re-register the same device during development. In this case,
```
(I need to update the style guide on voice)

follow these steps:

1. Delete the device from the UI ``Devices`` tab or with:

.. prompt:: bash host:~$

    fioctl device delete <device-name>

2. Stop ``aktualizr-lite`` on the device:

.. prompt:: bash device:~#

>     systemctl stop aktualizr-lite

Reviewer 2: you also should stop fioconfig.
```
systemctl stop fioconfig.path
systemctl stop fioconfig.service
```
